tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -2868,7 +2868,14 @@ public function testInvalidDocumentStructure()
         $this->assertEquals('Invalid document structure: Attribute "floatRange" has invalid format. Value must be a valid range between 1 and 1', $badFloatRange['body']['message']);
         $this->assertEquals('Invalid document structure: Attribute "probability" has invalid format. Value must be a valid range between 0 and 1', $badProbability['body']['message']);
         $this->assertEquals('Invalid document structure: Attribute "upperBound" has invalid format. Value must be a valid range between -9,223,372,036,854,775,808 and 10', $tooHigh['body']['message']);
-        $this->assertEquals('Invalid document structure: Attribute "lowerBound" has invalid format. Value must be a valid range between 5 and 9,223,372,036,854,775,807', $tooLow['body']['message']);
+
+        $max = '9,223,372,036,854,775,807';
+
+        if (PHP_VERSION_ID < 80300) {
+            $max = '9,223,372,036,854,775,808';
+        }
+
```

Comment:
why this check? cant we just use the PHP_INT_MAX constant or so?

Reply:
It's a PHP < 8.3 bug to have different value for the final number_format function
![image](https://github.com/user-attachments/assets/699791be-d0a7-4692-a444-6efd16a871ab)

Reply:
Guess it can be changed to this
```php
$max = \number_format(PHP_INT_MAX);
```